src/GLM.jl

```diff
@@ -1,12 +1,13 @@
 __precompile__()
 
 module GLM
+    using ArgCheck
```

Comment:
I'd prefer to keep the dependency list as light as possible, and IMO ArgCheck doesn't really provide sufficient benefit to warrant adding it as a dependency.